ui/packages/catalog-ui-search/src/main/webapp/js/OpenLayersGeometryUtils.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+/*jshint bitwise: false*/
+define([
+    'openlayers',
+    'properties',
+    './Common'
+], function (ol, properties, Common) {
```

Comment:
✏️ For new files, can we use a style more like
```
const ol = require('openlayers');
const properties = require('properties');
const Common = require('./Common');

module.exports = {
...
}
```

Reply:
We can use const there? I was afraid we were stuck with ES5 in this code.

Reply:
Yeah we can. There's a lot of legacy ES5 code, but we use Babel, so we _should_ be able to transpile any ES6 code to still be supported in IE10/IE11/etc